manager/README.md

```diff
@@ -12,6 +12,13 @@ The `manager` binary includes all of the controllers that this project defines b
 
 Beyond testing, you may run and debug the manager outside the cluster using the following instructions.
 
+### Build the fybrik environment
+Follow below Instructions to build the fybrik environment.
+
+1. Install Fybrik using the [Quick Start](https://fybrik.io/v0.5/get-started/quickstart/) guide.
+
```

Comment:
Please use `dev` version:  `https://fybrik.io/dev/get-started/quickstart/`

@pugangxa I would rename `Build the fybrik environment` to `Prepare the fybrik environment` and mention that the steps until `Install control plane` (excluded) section from the quick-start needs to be executed.

Reply:
OK, I saw there's already clause saying that so I just updated to make it clear.